src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/TextField/BitTextFieldDemo.razor

```diff
@@ -212,4 +212,18 @@
             }
         </ExamplePreview>
     </ComponentExampleBox>
+    <ComponentExampleBox Title="Trim TextField"
+                         HTMLSourceCode="@example14HTMLCode"
+                         CSharpSourceCode="@example14CSharpCode"
+                         ExampleId="example14">
+        <ExamplePreview>
+            <div class="example-desc">The trim property removes whitespace from both sides of a string.</div>
```

Comment:
Plz modify the desc:
`The trim property removes whitespaces from both sides of a string.`